### DISCUSSION TITLE
Please Provide API Support

### DISCUSSION THREAD

**Discussion by codeintelli** [dailydotdev/daily#883] [Feature Requests]
Hey Daily.dev team i hope you are doing well. It's a small request, could you please provide API so we can utilise your api to get all of your blogs and redirect users to your website when they click on blogs?

**Comment by idoshamun** [comment]
Hi @CodeIntelli,

We have plans to release an API for public use but nothing in the near future unfortunately

**Comment by capJavert** [comment]
**Update: daily.dev now has a public API** 🎉

This one's been asked for here for a long time, so: it's live, and it's
available to everyone. Generate a personal access token at
[daily.dev/settings/api](https://daily.dev/settings/api) and start calling it.

What's in v1:

- **Feeds** — your personalized For You feed, trending posts, filtering by tag or source
- **Posts** — full post details, AI summaries and comment threads
- **Search** — posts, tags and sources
- **Bookmarks** — read and manage your saved content
- **Notifications** — your activity feed and unread counts
- **Profile and reading streak** — your experience, tech stack and streak

📣 The announcement: https://daily.dev/posts/public-api-access-is-now-open-for-everyone-ljj8kh9sa
📚 Docs: https://docs.daily.dev/public-api/
